sfaira/data/dataloaders/loaders/d10_1038_s41591_018_0096_5/human_lung_2018_NA_Lambrechts_001.yaml

```diff
@@ -0,0 +1,43 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+dataset_wise:
+    author: "Lambrechts, Diether"
+    doi: "10.1038/s41591.018.0096.5"
+    download_url_data: "https://bit.ly/387ZM2s"
+    download_url_meta: "https://bit.ly/3dGyjGr"
+    normalization: "raw"
+    year: "2018"
+dataset_or_observation_wise:
+    age:
+    age_obs_key:
+    assay: "NA"
```

Comment:
this is not a valid assay, leave empty if you do not know it! In most cases, it is reported which method was used though.

Reply:
okay the init file was automatically generated, maybe good to specify somewhere what it needs to look like? or did I miss that documentation